packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -1809,11 +1809,14 @@ class Viewport {
       displayArea: true,
       zoom: true,
       pan: true,
+      flipHorizontal: true,
```

Comment:
Please represent this as rotation + flip horizontal to make the values canonical

Reply:
We will do this in another PR Bill